src/theme/utils.ts

```diff
@@ -24,13 +18,13 @@ const reduceColorShades = (arr: string[]) =>
     }, {} as generatedColorShades);
 
 const createShades = (func: (index: number) => string, numOfShades = 4) =>
-  new Array(numOfShades).fill(null).reduce((acc, _, index) => {
+  new Array(numOfShades).fill(null).reduce((acc, __, index) => {
```

Comment:
OCD kicking in? :joy:

Reply:
lol, linting required double underscore to define this as unused var